release.py

```diff
@@ -28,8 +28,12 @@
     f"dist/aiosmtpd-{ver_str}-py3-none-any.whl",
 ]
 
+RELEASE_TOOLKIT = ["setuptools", "wheel", "twine", "build"]
```

Comment:
`build` is the new build system created by PyPA that supports PEP 517.

Reply:
Yes. You need `build` for making dists and `twine` for linting them.

Reply:
Also, PyPI no longer supports GPG. It's possible to use Sigstore, though.

My PyPUG article has an example of that: https://packaging.python.org/en/latest/guides/publishing-package-distribution-releases-using-github-actions-ci-cd-workflows/.

Reply:
```suggestion
RELEASE_TOOLKIT = ["build", "twine"]
```